template/package.phone.appxmanifest

```diff
@@ -40,11 +40,9 @@
                          Description="CordovaApp"
                          ForegroundText="light"
                          BackgroundColor="transparent">
-        <m3:DefaultTile Wide310x150Logo="images\Wide310x150Logo.png"
-                        Square71x71Logo="images\Square71x71Logo.png">
+        <m3:DefaultTile  Square71x71Logo="images\Square71x71Logo.png">
```

Comment:
I am wondering the same...

But I would love the rest of the PR to be merged !